src/component/Page/CreatePeoplePage.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { NewPerson } from '../Forms/NewPerson';
+import { MoveBackBtn } from '../MoveBackBtn/MoveBackBtn';
+
+export const CreatePeoplePage: React.FC = () => {
```

Comment:
just opinion about your design 👀 
this input and button should be separated with some gap, cause at first I thought that you should use this button to submit the information inside the query input
![image](https://user-images.githubusercontent.com/57598825/146185278-b5f93e8d-4a8a-4e63-8c67-a4716b980083.png)